clients/web/src/stylesheets/widgets/accessWidget.styl

```diff
@@ -18,16 +20,15 @@
       color #d0b822
 
 .g-ac-list-title
-  background-color #666
+  background-color #888
```

Comment:
Not sure, but I get the feeling that `'#000'` is like the ALL-CAPS of the 90's - just too loud. Soft and friendly is where it's at.